qiskit/qpy/interface.py

```diff
@@ -284,6 +285,17 @@ def load(
             )
         )
 
+    config = user_config.get_config()
+
+    min_qpy_version = config.get("min_qpy_version")
+
+    if min_qpy_version is not None and data.qpy_version < min_qpy_version:
+
+        raise QiskitError(
```

Comment:
There is an argument that could be made for potentially adding a new exception class to differentiate this from other errors in loading. But I feel like we should at least be using the more specialized error class here rather than a more generic qiskit error 

```suggestion
        raise QpyError(
```